src/webpki.rs

```diff
@@ -305,5 +305,9 @@ pub struct TrustAnchor<'a> {
 
     /// The value of a DER-encoded NameConstraints, containing name
     /// constraints to apply to the trust anchor, if any.
-    pub name_constraints: Option<&'a [u8]>
+    pub name_constraints: Option<&'a [u8]>,
+
+    /// The root's expiry in seconds since the UNIX epoch. The anchor
+    /// is only considered when the validation time is before this value.
```

Comment:
Dividing seconds down to days is going to be annoying over 40-60 year periods, and I can't find anything in the time crate to help :(